daml.yaml

```diff
@@ -25,3 +25,5 @@ codegen:
   js:
     output-directory: daml.js
     npm-scope: daml.js
+build-options:
+  - --target=1.11
```

Comment:
We likely need to discuss the impact of moving to `daml-lf` version 1.11 as it may have impact on the API's calling it (I do agree we should move to it, we just need to ensure we're prepared for it).

Also, does the next version of `daml-lf` help you with your `Claims` work as I believe you had an issue with the size of the contract in the past (?) : 

> Daml-LF 1.12 is now included as an early access feature. Its main improvements are on performance, reducing transaction size through type erasure

Reply:
This is to support the new `Map`. I explained it in my first comment on the merge page.

Reply:
Yes, that's not my question - Using the new `Map` has an impact on the calling and receiving APIs and thus we'll need to decide to migrate to it now and update the UI or do it at some later point 

> TextMap itself continues to be supported as there are use-cases where it provides a nicer representation on the API than GenMap. Generic maps are represented as lists of tuples on the APIs. In contrast, TextMap is represented as a JSON object on the JSON API.

Reply:
I do take your point that this may not play nicely e.g. with DamlHub. So we should check that.

Reply:
> Its main improvements are on performance

So I haven't considered this yet. The 1.11 version is for functionality that I need it, not performance.

Reply:
> Yes, that's not my question - Using the new `Map` has an impact on the calling and receiving APIs and thus we'll need to decide to migrate to it now and update the UI or do it at some later point
> 
> > TextMap itself continues to be supported as there are use-cases where it provides a nicer representation on the API than GenMap. Generic maps are represented as lists of tuples on the APIs. In contrast, TextMap is represented as a JSON object on the JSON API.

Hmm - currently **all** `DA.Next.*` are throwing up deprecation warnings, contrary to what the above says. Where did you get that from?

Also, I see what you're saying now, in terms of the JSON map representation. I wasn't aware of that. I don't have an answer at this point.

Reply:
Here's a forum post on this: https://discuss.daml.com/t/clarify-deprecation-of-da-next-in-lf-1-11/2196

I'll now begin migrating `DA.Next`

Reply:
As discussed on slack, we've decided to do this in a separate PR. I will supress the warnings until we do so.